### DISCUSSION TITLE
Possibility to save a new post as a draft

### DISCUSSION THREAD

**Discussion by tremppu** [dailydotdev/daily#1536] [Feature Requests]
Could draft of unpublished post be saved so I could continue later without saving it somewhere else 🤔 
Edit: I noticed that it saved it now but did not save it earlier, there might be some bug 🤔

**Comment by idoshamun** [comment]
Hi @tremppu,

We don't have draft management but your current draft should be stored in the local storage to make sure you don't lose your work.
If it doesn't work for you please let me know
